src/skins/vector/views/molecules/MessageContextMenu.js

```diff
@@ -63,6 +81,13 @@ module.exports = React.createClass({
                 </div>
             );
         }
+        else {
+            redactButton = (
+                <div className="mx_ContextualMenu_field" onClick={this.onRedactClick}>
+                    Delete
```

Comment:
Delete is *not* Redact.  Delete means "please remove this message from my view of the room" (as we have in UC).  Redact means "please remove this message from all views of the room".  So I'd like it if we didn't mismatch the terminology for now and continue to call it Redact please :)